frontend/src/App.tsx

```diff
@@ -1,39 +1,42 @@
-import { useState } from "react";
+import { Outlet, Route, Routes } from "react-router-dom";
 
-import reactLogo from "./assets/react.svg";
-
-import viteLogo from "/vite.svg";
-import "./App.css";
-
-function App() {
-  const [count, setCount] = useState(0);
+import NavBar from "./components/NavBar.js";
+import AboutUs from "./pages/AboutUs.tsx";
```

Comment:
One last thing lets add the exports for the pages and nav like how Kathy and Jared did for the pages. [here](https://github.com/TritonSE/CCIDC-Application-Overhaul/blob/7-home-page/frontend/src/pages/index.ts) One index.ts file for each folder and update the import statements.